src/frontend/src/lib/providers/coingecko/Interface.ts

```diff
@@ -0,0 +1,44 @@
+import type { TokenId } from '$lib/types/token';
```

Comment:
can we prefix the interfaces' names to make those less general and more related to their purpose?